'dict object' has no attribute 'user_id'
